setup.py

```diff
@@ -8,7 +8,7 @@
 from setuptools import Extension, setup
 from setuptools.command.build_ext import build_ext
 
-__version__ = '0.7.2'
+__version__ = '0.7.3.alpha6'
```

Comment:
Just to confirm, idea is to keep version tag and this file in lockstep for now ?

(I can change this to use setuptools_scm later)

Reply:
Yes, that seemed the simplest thing for now. Could definitely do a more elaborate setup similar to `chroma` but might not even be necessary if this isn't being updated as frequently.